Form-Controls/index.html

```diff
@@ -1,28 +1,132 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Interpret requirements</h1>
+    </header>
+    <main>
+      <form enctype="multipart/form-data">
+        <article class="name-part">
+          <div class="part-1">
+            <label for="name">Name</label>
+            <input type="text" id="name" required autocomplete="given-name" />
+          </div>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+          <div class="part-1">
+            <label for="lastname">Last Name</label>
+            <input
+              type="text"
+              id="lastname"
+              required
+              autocomplete="last-name"
+            />
+          </div>
+        </article>
 
-</html>
\ No newline at end of file
+        <div class="gap">
+          <legend>Gender</legend>
+          <label for="male">Male</label>
+          <input
+            type="radio"
+            name="gender"
+            id="male"
+            required
+            autocomplete="gender"
+          />
+          <label for="female">Female</label>
+          <input
+            type="radio"
+            name="gender"
+            id="female"
+            required
+            autocomplete="gender"
+          />
```

Comment:
Great job on utilising a wide range of attributes(i.e `name` `autocomplete` Keep up the excellent work.